quiz.js

```diff
@@ -0,0 +1,75 @@
+console.log("Hello nurse");
+
+var enteredHeight = document.getElementById("howTallTree");
+var enteredCharacter = document.getElementById("whichCharacter")
+var button = document.getElementById("button");
+var enter = document.getElementById("button");
+var space = " ";
+var treeBuilder = "";
+var charWidth = 1;
+
+
+function growTree() {
+  enteredHeightValue=enteredHeight.value;
+  // console.log("entered Height Value", enteredHeightValue);
+  enteredCharacterValue=enteredCharacter.value;
+  // console.log("enteredCharacterValue", enteredCharacterValue);
+  var spaces = 1;
+  // work on growTree function here
+
+  treeBuilder = space.repeat(enteredHeightValue) + enteredCharacterValue;
+    console.log("treeBuilder", treeBuilder);
+
+  for (var i=0; i<enteredHeightValue; i++) {
+    charWidth += 2;
+    treeBuilder = space.repeat(enteredHeightValue - parseInt(spaces)) + enteredCharacterValue.repeat(parseInt(charWidth));
+    console.log("treeBuilder", treeBuilder);
+    spaces+=1;
+  }
+    // treeBuilder = space.repeat(enteredHeightValue) + enteredCharacterValue
+    // console.log("treeBuilder", treeBuilder);
+    // treeBuilder = space.repeat(enteredHeightValue - parseInt(width)) + enteredCharacterValue.repeat(3);
+    // console.log("treeBuilder", treeBuilder);
+    // treeBuilder = space.repeat(enteredHeightValue - parseInt(2)) + enteredCharacterValue.repeat(5);
+    // console.log("treeBuilder", treeBuilder);
+    // treeBuilder = space.repeat(enteredHeightValue - parseInt(3)) + enteredCharacterValue.repeat(7);
+    // console.log("treeBuilder", treeBuilder);
+
+
+  // for (var i=0; i<enteredHeightValue; i++) {
+  //   // add enteredHeightValue number of space then add enteredCharacterValue
+  //   treeBuilder += space.repeat(enteredHeightValue) + enteredCharacterValue;
+
+
+  //   console.log("treeBuilder", treeBuilder);
+  //   // subtract two from enteredHeightValue and add two enteredCharacterValue
+  // }
+
+
+
+
+
+  if (enteredHeightValue.length === 0) {
+    // console.log("enter a height value");
```

Comment:
ah, okay, cool, i didn’t see your note at the bottom, i’ll do that and resubmit


> On Jun 3, 2016, at 9:47 AM, Joe Shepherd <notifications@github.com> wrote:
> 
> In quiz.js <https://github.com/nss-day-cohort-14/static-web-js-quiz-vaughnwalters/pull/1#discussion_r65719462>:
> 
> > +
> > +  // for (var i=0; i<enteredHeightValue; i++) {
> > +  //   // add enteredHeightValue number of space then add enteredCharacterValue
> > +  //   treeBuilder += space.repeat(enteredHeightValue) + enteredCharacterValue;
> > +
> > +
> > +  //   console.log("treeBuilder", treeBuilder);
> > +  //   // subtract two from enteredHeightValue and add two enteredCharacterValue
> > +  // }
> > +
> > +
> > +
> > +
> > +
> > +  if (enteredHeightValue.length === 0) {
> > +    // console.log("enter a height value");
> Commented out code
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/nss-day-cohort-14/static-web-js-quiz-vaughnwalters/pull/1/files/c741e3933fdf8917ebbdd4123890646a365127e8#r65719462>, or mute the thread <https://github.com/notifications/unsubscribe/ASWbr2c5855XOyXtCPyOCe8R8d4F-U-wks5qID6IgaJpZM4Ir0ch>.
>